consensus/istanbul/backend/api.go

```diff
@@ -474,6 +477,90 @@ func (api *APIExtension) GetBlockWithConsensusInfoByHash(blockHash common.Hash)
 	return api.makeRPCBlockOutput(block, cInfo, block.Transactions(), receipts), nil
 }
 
+func (api *APIExtension) GetBlsInfos(number rpc.BlockNumber) (map[string]interface{}, error) {
+	kip113Addr, err := api.GetActiveAddressFromRegistry(system.Kip113Name, number)
+	if err != nil {
+		return nil, err
+	}
+
+	var bn *big.Int
+	if number == rpc.LatestBlockNumber || number == rpc.PendingBlockNumber {
+		bn = big.NewInt(api.chain.CurrentBlock().Number().Int64())
+	} else {
+		bn = big.NewInt(number.Int64())
+	}
+
+	backend := backends.NewBlockchainContractBackend(api.chain, nil, nil)
+	infos, err := system.ReadKip113All(backend, kip113Addr, bn)
+	if err != nil {
+		return nil, err
+	}
+
+	blsInfos := make(map[string]interface{})
+	for addr, info := range infos {
+		// make publicKey, Pop to hexadecimal string
+		blsInfos[addr.Hex()] = map[string]interface{}{
+			"publicKey": hex.EncodeToString(info.PublicKey),
+			"pop":       hex.EncodeToString(info.Pop),
+			"verifyErr": info.VerifyErr,
+		}
+	}
+	return blsInfos, nil
+}
+
+func (api *APIExtension) GetAllRecordsFromRegistry(name string, number rpc.BlockNumber) ([]interface{}, error) {
+	var bn *big.Int
+	if number == rpc.LatestBlockNumber || number == rpc.PendingBlockNumber {
+		bn = big.NewInt(api.chain.CurrentBlock().Number().Int64())
+	} else {
+		bn = big.NewInt(number.Int64())
+	}
+
+	if api.chain.Config().IsRandaoForkEnabled(bn) {
+		backend := backends.NewBlockchainContractBackend(api.chain, nil, nil)
+		records, err := system.ReadAllRecordsFromRegistry(backend, name, bn)
+		if err != nil {
+			return nil, err
+		}
+
+		if len(records) == 0 {
+			return nil, errors.New("no records for " + name)
```

Comment:
```suggestion
			return nil, fmt.Errorf("%s has not been registered", name)
```
to match with KIP-149.

Reply:
Applied at https://github.com/klaytn/klaytn/pull/2111/commits/9c5c06fef2d1fcdadce301ee0f67d211c4225026